### PR TITLE
fix accept header application/json for transactionListAsString method

### PR DESCRIPTION
**Pull request by spetrychenko**

### CODE REVIEW

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java [hunk 1]
```diff
@@ -212,20 +219,29 @@ private ObjectNode createEmptyBody() {
...
         return (ResponseEntity<T>) client.getTransactionListAsString(accountId, dateFrom, dateTo, entryReferenceFrom, bookingStatus, deltaList, withBalance, requestHeaders.toMap());
     }
 
     @Override
     public Response<String> getTransactionListAsString(String accountId, RequestHeaders requestHeaders, RequestParams requestParams) {
         ResponseEntity<String> responseEntity = getTransactionListFromClient(accountId, requestParams, requestHeaders);
 
+        String body;
+
+        if (responseEntity.getBody() instanceof String) {
```
Comment (artemsnisarenko) [inline xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java]: Could you explain why you think this instance of check is necessary and when body is going to be a non-string type?

File: xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        when(client.getTransactionListAsString(any(), any(), any(), anyString(), any(), anyBoolean(), anyBoolean(), anyMap())).thenReturn(buildResponseEntity());
+
+        Response<String> response = service.getTransactionListAsString(
+            "accountId",
+            RequestHeaders.fromMap(buildHeaders()),
+            RequestParams.fromMap(buildRequestParams())
+        );
+
+        assertThat("{\"bookingStatus\":\"booked\"}", is(response.getBody()));
```
Comment (artemsnisarenko) [inline xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationServiceTest.java]: booking status is not a transaction report field, how is this working?

**Comment by artemsnisarenko** [comment]
Please wrap the lines that exceed 120 characters.

File: xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java (hunk 1 above)
Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java]: you are right, removed

File: xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationServiceTest.java (hunk 2 above)
Comment (spetrychenko) [reply xs2a-adapter-service-remote/src/test/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationServiceTest.java]: thanks, updated

**Pull request merged**